prov/verbs/src/verbs_domain.c

```diff
@@ -60,12 +58,7 @@ static int fi_ibv_domain_bind(struct fid *fid, struct fid *bfid, uint64_t flags)
 			domain->eq_flags = flags;
 			break;
 		case FI_EP_DGRAM:
-			dgram_eq = container_of(bfid, struct fi_ibv_dgram_eq,
-						util_eq.eq_fid);
-			if (!dgram_eq)
-				return -FI_EINVAL;
-			return ofi_domain_bind_eq(&domain->util_domain,
-						  &dgram_eq->util_eq);
+			return -FI_EINVAL;
```

Comment:
Is EQ for DGRAM endpoints being removed?

Reply:
Yes, EQ, CNTR has been removed, because they are not needed.
Once they are needed, we add them. Is it OK?

Reply:
Sure. Please add this info in the commit message.

Reply:
> Please add this info in the commit message

Sure